From d08ac4e89b6fda4b2d72895dbaad27254e950b07 Mon Sep 17 00:00:00 2001
From: Charles Stevens <charles.stevens@compasses.com>
Date: Wed, 2 Dec 2020 17:43:08 -0600
Subject: [PATCH 35/44] RSOM-186 - Fix AVB TXC refclock reference

arm64: dts: Rename symbol for SOM versaclock to match actual part.

Signed-off-by: Charles Stevens <charles.stevens@compasses.com>
---
 arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi | 4 ++--
 arch/arm64/boot/dts/renesas/beacon-renesom-som.dtsi       | 6 +++---
 arch/arm64/boot/dts/renesas/r8a774a1-beacon-rzg2m-kit.dts | 4 ++--
 arch/arm64/boot/dts/renesas/r8a774b1-beacon-rzg2n-kit.dts | 4 ++--
 arch/arm64/boot/dts/renesas/r8a774e1-beacon-rzg2h-kit.dts | 4 ++--
 5 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
index 7b2a4445bf2f..48d3081c332e 100755
--- a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
+++ b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
@@ -290,12 +290,12 @@
 &ehci0 {
 	dr_mode = "otg";
 	status = "okay";
-	clocks = <&cpg CPG_MOD 703>, <&cpg CPG_MOD 704>, <&versaclock5 3>;
+	clocks = <&cpg CPG_MOD 703>, <&cpg CPG_MOD 704>, <&versaclock6_som 3>;
 };
 
 &ehci1 {
 	status = "okay";
-	clocks = <&cpg CPG_MOD 703>, <&cpg CPG_MOD 704>, <&versaclock5 3>;
+	clocks = <&cpg CPG_MOD 703>, <&cpg CPG_MOD 704>, <&versaclock6_som 3>;
 };
 
 &hdmi0 {
diff --git a/arch/arm64/boot/dts/renesas/beacon-renesom-som.dtsi b/arch/arm64/boot/dts/renesas/beacon-renesom-som.dtsi
index 172327da3de0..599b3d874c37 100644
--- a/arch/arm64/boot/dts/renesas/beacon-renesom-som.dtsi
+++ b/arch/arm64/boot/dts/renesas/beacon-renesom-som.dtsi
@@ -88,7 +88,7 @@
 	pinctrl-names = "default";
 	phy-handle = <&phy0>;
 	phy-mode = "rgmii-txid";
-	clocks = <&cpg CPG_MOD 812>, <&versaclock5 4>;
+	clocks = <&cpg CPG_MOD 812>, <&versaclock6_som 4>;
 	clock-names = "etheravb", "txc_refclk";
 	status = "okay";
 
@@ -324,14 +324,14 @@
 		reg = <0x51>;
 	};
 
-	versaclock5: versaclock_som@6a {
+	versaclock6_som: versaclock_som@6a {
 		compatible = "idt,5p49v6965";
 		reg = <0x6a>;
 		#clock-cells = <1>;
 		clocks = <&x304_clk>;
 		clock-names = "xin";
 		/* du_dotclkin0, du_dotclkin2, usb_extal, avb_txcrefclk */
-		assigned-clocks =	<&versaclock5 1>, <&versaclock5 2>, <&versaclock5 3>, <&versaclock5 4>;
+		assigned-clocks =	<&versaclock6_som 1>, <&versaclock6_som 2>, <&versaclock6_som 3>, <&versaclock6_som 4>;
 		assigned-clock-rates =	<33333333>, <33333333>, <50000000>, <125000000>;
 
 		OUT1 {
diff --git a/arch/arm64/boot/dts/renesas/r8a774a1-beacon-rzg2m-kit.dts b/arch/arm64/boot/dts/renesas/r8a774a1-beacon-rzg2m-kit.dts
index 814cad204caf..dfe0ab257a63 100644
--- a/arch/arm64/boot/dts/renesas/r8a774a1-beacon-rzg2m-kit.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774a1-beacon-rzg2m-kit.dts
@@ -35,9 +35,9 @@
 	clocks = <&cpg CPG_MOD 724>,
 		<&cpg CPG_MOD 723>,
 		<&cpg CPG_MOD 722>,
-		<&versaclock5 1>,
+		<&versaclock6_som 1>,
 		<&x302_clk>,
-		<&versaclock5 2>;
+		<&versaclock6_som 2>;
 	clock-names = "du.0","du.1", "du.2",
 		"dclkin.0", "dclkin.1", "dclkin.2";
 };
diff --git a/arch/arm64/boot/dts/renesas/r8a774b1-beacon-rzg2n-kit.dts b/arch/arm64/boot/dts/renesas/r8a774b1-beacon-rzg2n-kit.dts
index facbb7e7bf74..60c4c15c138d 100644
--- a/arch/arm64/boot/dts/renesas/r8a774b1-beacon-rzg2n-kit.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774b1-beacon-rzg2n-kit.dts
@@ -30,9 +30,9 @@
 	clocks = <&cpg CPG_MOD 724>,
 		<&cpg CPG_MOD 723>,
 		<&cpg CPG_MOD 721>,
-		<&versaclock5 1>,
+		<&versaclock6_som 1>,
 		<&x302_clk>,
-		<&versaclock5 2>;
+		<&versaclock6_som 2>;
 	clock-names = "du.0", "du.1", "du.3",
 		"dclkin.0", "dclkin.1", "dclkin.3";
 };
diff --git a/arch/arm64/boot/dts/renesas/r8a774e1-beacon-rzg2h-kit.dts b/arch/arm64/boot/dts/renesas/r8a774e1-beacon-rzg2h-kit.dts
index 07dc5871f3f1..25692027769f 100644
--- a/arch/arm64/boot/dts/renesas/r8a774e1-beacon-rzg2h-kit.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774e1-beacon-rzg2h-kit.dts
@@ -35,9 +35,9 @@
 	clocks = <&cpg CPG_MOD 724>,
 		<&cpg CPG_MOD 723>,
 		<&cpg CPG_MOD 721>,
-		<&versaclock5 1>,
+		<&versaclock6_som 1>,
 		<&x302_clk>,
-		<&versaclock5 2>;
+		<&versaclock6_som 2>;
 	clock-names = "du.0", "du.1", "du.3",
 		"dclkin.0", "dclkin.1", "dclkin.3";
 };
-- 
2.17.1

